ex01.ex

```diff
@@ -60,7 +62,11 @@ defmodule Ex01 do
   # no explicit + operators in your function                          #
   #####################################################################
 
-  sum3a = your_anonymous_function(1, 2, 3)
+  sum3a = fn (a, b, c) ->
+    tempSum = sum2a.(a, b)
```

Comment:
## -1

Why did you use a temporary variable here and not in the next one?

In fact, this problem has an elegant solution using a pipeline:

~~~ elixir
  sum3a = fn a, b, c -> a |> sum2a.(b) |> sum2a.(c) end
~~~